frontend/pages/admin/UserManagementPage/components/UserForm/UserForm.tsx

```diff
@@ -424,189 +468,202 @@ const UserForm = ({
+        name="authentication-type"
+        onChange={onSsoDisable}
+        /** Note: This was a checkbox that is now a radio button, waiting on
+         * product to confirm if we're adding help text to radio buttons as
+         * it's not in Figma design system, the Figma here, or in the Radio
+         * component, but the helpText already existed on the checkbox version
+         */
+        // helpText={
+        //   canUseSso ? (
+        //     <p className={`${baseClass}__sso-input sublabel`}>
+        //       Password authentication will be disabled for this user.
+        //     </p>
+        //   ) : (
+        //     <span className={`${baseClass}__sso-input sublabel-nosso`}>
+        //       This user previously signed in via SSO, which has been
+        //       globally disabled.{" "}
+        //       <button
+        //         className="button--text-link"
+        //         onClick={onSsoDisable}
+        //       >
+        //         Add password instead
+        //         <Icon
+        //           name="chevron-right"
+        //           color="core-fleet-blue"
+        //           size="small"
+        //         />
+        //       </button>
+        //     </span>
+        //   )
+        // }
+      />
+    </div>
+  );
+
+  const renderPasswordSection = () => (
+    <div className={`${baseClass}__${isNewUser ? "" : "edit-"}password`}>
+      <InputField
+        label="Password"
+        error={errors.password}
+        name="password"
+        onChange={onInputChange("password")}
+        placeholder={isNewUser ? "Password" : "••••••••"}
+        value={formData.password || ""}
+        type="password"
+        helpText="12-48 characters, with at least 1 number (e.g. 0 - 9) and 1 symbol (e.g. &*#)."
+        blockAutoComplete
+        tooltip={
+          isNewUser ? (
+            <>
+              This password is temporary. This user will be asked to set a new
+              password after logging in to the Fleet UI.
+              <br />
+              <br />
+              This user will not be asked to set a new password after logging in
+              to fleetctl or the Fleet API.
+            </>
+          ) : undefined
+        }
+      />
+    </div>
+  );
+
+  // 2fa option shows on premium tier or if previously set to true before downgrading to free
+  const renderTwoFactorAuthenticationOption = () => (
+    <div className="form-field">
+      {/* Renders missing password heading when inviting a user */}
+      {formData.newUserType === NewUserType.AdminInvited && (
+        <div className="form-field__label">Password</div>
       )}
-      {!isPremiumTier && renderGlobalRoleForm()}
-
-      <div className="modal-cta-wrap">
-        <Button
-          type="submit"
-          variant="brand"
-          onClick={onFormSubmit}
-          className={`${submitText === "Create" ? "create" : "save"}-loading
-          `}
-          isLoading={isUpdatingUsers}
+      <Checkbox
```

Comment:
Pretty sure we need to disable this if we don't have a mailer set up.

Reply:
done! ty for working with product to find all the nuances